e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js

```diff
@@ -33,6 +33,16 @@ Scenario('Judiciary adds allocated judge', async (I, caseViewPage, allocatedJudg
   caseViewPage.selectTab(caseViewPage.tabs.casePeople);
   I.seeInTab(['Allocated Judge', 'Judge or magistrate\'s title'], 'Her Honour Judge');
   I.seeInTab(['Allocated Judge', 'Last name'], 'Moley');
+  I.seeInTab(['Allocated Judge', 'Email Address'], 'moley@example.com');
+
+  await I.navigateToCaseDetailsAs(config.swanseaLocalAuthorityUserOne, caseId);
+
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  I.seeInTab(['Allocated Judge', 'Judge or magistrate\'s title'], 'Her Honour Judge');
+  I.seeInTab(['Allocated Judge', 'Last name'], 'Moley');
+  I.dontSeeInTab(['Allocated Judge', 'Email Address']);
```

Comment:
Minor but is the email address suppose to be title case?

Reply:
> Minor but is the email address suppose to be title case?

https://tools.hmcts.net/jira/browse/FPLA-1664 - it was developed as part of this story and is approved. Do you think i should change it?

Reply:
As discussed maybe out of scope for this story :(